doc/newsfragments/reload_schedule_task_target.change.rst

```diff
@@ -0,0 +1 @@
+* Remove interactive reload limitation: the scheduled task target should be supported.
```

Comment:
Improvement for interactive mode reload feature (link to how reload works): can now pick up changes to the test added by plan.schedule().